bot/utils.py

```diff
@@ -10,3 +15,28 @@ def build_menu(buttons,
     if footer_buttons:
         menu.append([footer_buttons])
     return menu
+
+
+def convert_dcr(dcr_amount: float, target_currency: str):
+    dcrdata_response = requests.get("https://dcrdata.decred.org/api/exchanges")
```

Comment:
Good for now, but when bot get more users, maybe is better to create an updater for this endpoint and store response in db.

Reply:
Yes, saving a timestamp. And if there's another request for this command before some interval of `x` seconds we return the last value.

Want this now or should we postpone?

Reply:
Create a new issue